cloud/scope/machine.go

```diff
@@ -155,6 +155,19 @@ func (m *MachineScope) CreateMachine() (*vpcv1.Instance, error) {
 			}
 			instancePrototype.Keys = append(instancePrototype.Keys, key)
 		}
+	} else if m.IBMVPCMachine.Spec.SSHKeysRef != nil {
```

Comment:
but we discussed about using both values `SSHKeys SSHKeysRef` and right?

Reply:
Fixed.